rllib/tests/test_nested_action_spaces.py

```diff
@@ -117,7 +117,7 @@ def test_nested_action_spaces(self):
                         ioctx.config["input_config"]["paths"], ioctx
                     )
                     config["input_config"] = {"paths": config["output"]}
-                    del config["output"]
+                    del config.output
```

Comment:
Nit: I don't think we should `del` this here, since it's now a property. Just set it to `None`?